scripts/globals/spells/bluemagic/pinecone_bomb.lua

```diff
@@ -0,0 +1,66 @@
+-----------------------------------------
+-- Spell: Pinecone Bomb
+-- Additional effect: sleep. Duration of effect varies with TP
+-- Spell cost: 48 MP
+-- Monster Type: Plantoids
+-- Spell Type: Physical (Piercing)
+-- Blue Magic Points: 2
+-- Stat Bonus: STR+1
+-- Level: 36
+-- Casting Time: 2.5 seconds
+-- Recast Time: 26.5 seconds
+-- Skillchain property: Liquefaction (can open Scission or Fusion)
+-- Combos: None
+-----------------------------------------
+
+require("scripts/globals/magic");
+require("scripts/globals/status");
+require("scripts/globals/bluemagic");
+
+-----------------------------------------
+-- onMagicCastingCheck
+-----------------------------------------
+
+function onMagicCastingCheck(caster,target,spell)
+    return 0;
+end;
+
+-----------------------------------------
+-- OnSpellCast
+-----------------------------------------
+
+local resist = duration*applyResistance();
```

Comment:
Because I am a noob that barely understands what I  am doing. Seems to be a relic from whatever spell I copied the code from. There is also a resist inside. I will delete when I get home.